core/msg.c

```diff
@@ -242,7 +243,7 @@ int msg_send_bus(msg_t *m, msg_bus_t *bus)
     int count = 0;
 
     m->sender_pid = (in_irq ? KERNEL_PID_ISR : thread_getpid())
-                  | MSB_BUS_PID_FLAG;
+                    | MSB_BUS_PID_FLAG;
```

Comment:
Is this really what we want?

Reply:
I just used `uncrustify` and @kaspar030 said, changes made by `uncrustify` are non-negotiable.

Reply:
+ I actually would prefer it as proposed by `uncrustify` (aligning RHS of `=` operator).